### PR TITLE
[FIX] web_edtor: fix for pillow < 8.0

### PR DESCRIPTION
**Pull request by Xavier-Do**
3aa3a24c introduced a regression since textbbox is not available in Pillow < 8.0. 

Since focal has a lower version, this is breaking on odoosh and in PureFocal builds.

Fixing it by making the call conditional.

closes #162256

### CODE REVIEW

File: addons/web_editor/tests/play_cropped.png
Comment (Mtaylorr) [inline addons/web_editor/tests/play_cropped.png:1]: I just have one question , why do we need to include the image for the cropped version too ? is there a reason behind it ?

**Review by Mtaylorr — APPROVED** [review]

File: addons/web_editor/tests/play_cropped.png
Comment (Xavier-Do) [reply addons/web_editor/tests/play_cropped.png:1]: Not really, was a way to test the assertion and document the unwanted result, but I will remove it, it has a low value.

**Comment by Xavier-Do** [comment]
@odoo/rd-security 

Catching the `Image.open` from Pillow, done on an odoo `file_open` with a static path in a test

**Review by jorv-odoo — APPROVED** [review]
Did a quick functional test simulation the issue with the email marketing social media font icon (e.g. opw-3926192).
In local, Odoo 15:
* pillow==7.0.0 -> icons render correctly
* pillow==9.0.1 -> icons render correctly

So as far as the bug is concerned that lead me here, everything works. 
(Not familiar enough with this part of the code to know if it might affect other parts)

**Comment by Xavier-Do** [comment]
@robodoo retry

**Pull request closed without merging**